src/watertap_contrib/reflo/costing/tests/test_reflo_watertap_costing_package.py

```diff
@@ -332,7 +332,7 @@ def build_heat_and_elec_gen():
     #### TREATMENT BLOCK
     m.fs.treatment = Block()
     m.fs.treatment.costing = TreatmentCosting()
-    m.fs.treatment.costing.base_currency = pyunits.USD_2002
+    m.fs.treatment.costing.base_currency = pyunits.USD_2023
```

Comment:
Shouldn't this be set as default in the costing package and propagate down to the two different Costing blocks?

Reply:
Done